tests/hypervisor/conftest.py

```diff
@@ -204,9 +204,6 @@ def kubevirt_assertion():
     Collect all the assertion info for kubevirt to this fixture
     :return:
     """
-    disable_error = "Failed to connect socket to '/var/run/libvirt/libvirt-sock-ro'"
-    if "RHEL-9" in RHEL_COMPOSE:
-        disable_error = "Cannot use direct socket mode if no URI is set"
```

Comment:
remove the if branch because it's for the rhel-8 and rhel-9, now the main branch will only support the rhel-9 and rhel-10, and the disable_error is same for the 9 and 10